allure-web/src/components/tree/TreeView.js

```diff
@@ -14,9 +14,13 @@ class TreeView extends View {
         this.state = state;
         this.baseUrl = baseUrl;
         this.tabName = tabName;
-        this.statusesSelect = new StatusToggleView();
+        this.statusesKey = tabName + 'visibleStatuses';
```

Comment:
Maybe it would be better to make this key with dot, like `tabName + '.visibleStatuses'`.
Not a big change, but better to read in debug